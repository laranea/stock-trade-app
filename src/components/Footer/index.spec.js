import React from 'react';
import { mount } from 'enzyme';

import Footer from './';

describe('Footer component', () => {
  it('should match snapshot', () => {
    const component = mount(<Footer />);
    expect(component).toMatchSnapshot();
  });
});
