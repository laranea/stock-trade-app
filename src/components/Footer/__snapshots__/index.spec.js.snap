// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should match snapshot 1`] = `
<Footer>
  <styled.header>
    <StyledComponent
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": true,
            "lastClassName": "bVruJa",
            "rules": Array [
              "
  position: fixed;
  bottom: 0;
  width: 100%;
  height: 40px;
  padding: 0 5rem;
  background: #ffffff;
  border-top: 2px solid #dddddd;
",
            ],
          },
          "displayName": "styled.header",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "header",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
    >
      <header
        className="sc-bdVaJa bVruJa"
      />
    </StyledComponent>
  </styled.header>
</Footer>
`;
